actors/src/main/scala/zio/actors/Utils.scala

```diff
@@ -0,0 +1,44 @@
+package zio.actors
+
+import java.io.File
+
+import zio.Task
+import zio.config.ConfigDescriptor
+import zio.config.ConfigDescriptor._
+import zio.config.typesafe.TypesafeConfig
+
+private[actors] object Utils {
```

Comment:
These aren't really "utils", they're tied to the configuration, hence any name on that line should be a better choice.

Reply:
How about `ConfigUtils` or `ConfigAccess`?

Reply:
Or `ActorsConfig`?